app/services/insert-fixed-option.js

```diff
@@ -0,0 +1,24 @@
+import Ember from "ember";
```

Comment:
Maybe worth moving that into the subform service ?  I have trouble understanding the name `insert-fixed-option` 🤔